rl4co/models/zoo/mdam/decoder.py

```diff
@@ -54,8 +61,14 @@ def __init__(
             -1, 1
         )  # Placeholder should be in range of activations
 
+        if context_embedding is None:
+            context_embedding = env_context_embedding(env_name, {"embedding_dim": embedding_dim})
+        else:
+            context_embedding = context_embedding
+        
         self.context = [
-            env_context_embedding(env_name, {"embedding_dim": embedding_dim})
+            # env_context_embedding(env_name, {"embedding_dim": embedding_dim})
```

Comment:
This kind of won't used code could be directly removed, since we have Git it would be easy to track. Don't worry.